library/Fission/Error.hs

```diff
@@ -0,0 +1,29 @@
+module Fission.Error
+  ( openLeft
+  , relaxedLeft
+  , fromMaybe
+  , fromMaybe'
+  , module Fission.Error.Types
+  ) where
+
+import Data.WorldPeace
```

Comment:
why import these separately instead of `Fission.Prelude`?

Reply:
Ah, good point! I had a module cycle at one point, and removed `Fission.Prelude`. Will put that back as it was 👍